src/app/backend-api.service.ts

```diff
@@ -1859,6 +1906,116 @@ export class BackendApiService {
     return this.signAndSubmitTransaction(endpoint, request, ReaderPublicKeyBase58Check);
   }
 
+  CreatePostAssociation(
+    Endpoint: string,
+    TransactorPublicKeyBase58Check: string,
+    PostHashHex: string,
+    AssociationType: AssociationType,
+    AssociationValue: AssociationValue
+  ): Observable<any> {
+    const request = this.post(Endpoint, BackendRoutes.RoutePathCreatePostAssociation, {
+      TransactorPublicKeyBase58Check,
+      PostHashHex,
+      AppPublicKeyBase58Check: BackendApiService.DIAMOND_APP_PUBLIC_KEY,
+      AssociationType,
+      AssociationValue,
+      ExtraData: {},
+      MinFeeRateNanosPerKB: 1000,
+      TransactionFees: [],
+    });
+
+    return this.signAndSubmitTransaction(Endpoint, request, TransactorPublicKeyBase58Check);
+  }
+
+  DeletePostAssociation(
+    Endpoint: string,
+    TransactorPublicKeyBase58Check: string,
+    AssociationID: string
+  ): Observable<any> {
+    const request = this.post(Endpoint, BackendRoutes.RoutePathDeletePostAssociation, {
+      TransactorPublicKeyBase58Check,
+      AssociationID,
+      MinFeeRateNanosPerKB: 1000,
+      TransactionFees: [],
+    });
+
+    return this.signAndSubmitTransaction(Endpoint, request, TransactorPublicKeyBase58Check);
+  }
+
+  GetAllPostAssociations(
+    Endpoint: string,
+    PostHashHex: string,
+    AssociationType: AssociationType,
+    TransactorPublicKeyBase58Check?: string,
+    AssociationValue?: AssociationValue,
+    total: number = 0
+  ) {
+    const ASSOCIATIONS_PER_REQUEST_LIMIT = 100;
+    let receivedItems = [];
+
+    const fetchAssociationsChunk = (LastSeenAssociationID?: string) => {
+      return this.GetPostAssociations(
+        Endpoint,
+        PostHashHex,
+        AssociationType,
+        TransactorPublicKeyBase58Check,
+        AssociationValue,
+        LastSeenAssociationID,
+        ASSOCIATIONS_PER_REQUEST_LIMIT
```

Comment:
There was a suggestion not to include an `AppPublicKeyBase58Check` when we request the associations, no matter if they come from different application. Let's discuss in details if that seems wrong